frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/BaseTraverseAction.java

```diff
@@ -96,7 +96,7 @@ public void actionPerformed(AnActionEvent anActionEvent) {
             getString("action.GitMachete.BaseTraverseAction.dialog.traverse-approval.title"),
             getString("action.GitMachete.BaseTraverseAction.dialog.traverse-approval.text.HTML"),
             SHOW_TRAVERSE_INFO,
-            /* myHeight */ 100);
+            /* myHeight */ 120);
```

Comment:
Huh just a 20% increase will be enough? 🤔

Reply:
it's enough to me but I will perform beta testing 😆